include/z64save.h

```diff
@@ -662,9 +662,9 @@ typedef enum {
 #define INFTABLE_76 0x76
 #define INFTABLE_77 0x77
 #define INFTABLE_TALKED_TO_TALON_IN_RANCH_HOUSE 0x7E
-#define INFTABLE_84 0x84
-#define INFTABLE_85 0x85
-#define INFTABLE_8B 0x8B
+#define INFTABLE_TALKED_TO_MALON_FIRST_TIME 0x84
+#define INFTABLE_TOLD_EPONA_IS_SCARED 0x85
+#define INFTABLE_ENTERED_HYRULE_CASTLE_SCENE 0x8B
 #define INFTABLE_8C 0x8C
```

Comment:
`INFTABLE_ENTERED_HYRULE_CASTLE_SCENE` should probably still be named centered around malon somehow. With the current name, youd expect some higher level code to be watching for link to enter hyrule castle *in general*. But really its "malon spawned in hyrule castle for the first time"

I know in practice malon spawning in hyrule castle is always going to coincide with link going there the first time. But I think its an important distinction still